sootup.callgraph/src/main/java/sootup/callgraph/CallGraph.java

```diff
@@ -86,6 +86,9 @@ boolean containsCall(
    */
   int callCount();
 
+  /** This method converts the call graph object into dot format and write it to a string file. */
+  void convertToDotFormatAndFileWrite(String filePath);
```

Comment:
exportAsDot() is a better name

Reply:
is there a method which returns it as String (as well)?

Reply:
Now we changed the method to just return a dot formatted string and the file write feature is removed.